airflow/ui/src/layouts/Nav/Nav.tsx

```diff
@@ -59,7 +59,7 @@ export const Nav = () => {
         <NavButton
           icon={<FiCornerUpLeft size="1.75rem" />}
           title="Legacy UI"
-          to={import.meta.env.VITE_LEGACY_API_URL}
+          to="http://0.0.0.0:28080/home"
```

Comment:
Mhm, 0.0.0.0 is not a valid address - Why are you sure this is fixing the button?
I assume per default to link is broken if the ENV file is not adjusted.